#161290848 Setup Empty React Project

ajimae opened the pull request:
#### What does this PR do?
* Setup an empty react boilerplate


#### Description of Task to be completed?
- Setup Webpack 4
- Setup babel version 7
- Configure Webpack for production and development
- Modify npm script to run build and start the server
- Setup eslint Airbnb style guide
- Modify npm base script to run linting from the command line


#### How should this be manually tested?
- Clone the repo to local machine, run `npm run start` and visit `http://localhost:8080`


#### Any background context you want to provide?
None


#### What are the relevant pivotal tracker stories?
[#161290848](https://www.pivotaltracker.com/story/show/161290848)


#### Screenshot
<img width="1420" alt="screen shot 2018-11-30 at 10 02 53 am" src="https://user-images.githubusercontent.com/32770340/49279635-4f6be200-f488-11e8-9380-796a3d37f1ee.png">

ovieokeh reviewed (changes requested):
Running `npm start` and opening `dist/index.html` displays a blank file. Shouldn't it display an image and the Captain Kirk dialogue instead?

Commit:
Chore(ReactProject) Setup Empty React Project:

- Setup Webpack 4
- Setup babel version 7
- Configure Webpack for production and development
- Modify npm script to run build and start server
- Setup eslant airbnb style guide
- Modify npm base script to run linting from the command line

[Finishes #161290848]
